tests/api/EventCest.php

```diff
@@ -1,28 +1,468 @@
+            $ip = '188.163.122.80';
+            $meta = [];
+            $date = (new \DateTime())->format('Y-m-d');
+
+            $I->sendPost('/event', [
+                'data' => [
+                    'type' => 'events',
+                    'attributes' => [
+                        'temporary_user_id' => $temporaryUserId,
+                        'user_id' => $userId,
+                        'event' => $event,
+                        'value' => '',
+                        'tag' => $tag,
+                        'referrer' => $referrer,
+                        'ip' => $ip,
+                        'country_code' => 'ua',
+                        'meta' => $meta,
+                        'created_at' => $date
+                    ],
+                    'relationships' => [
+                        'owner' => [
+                            'data' => [
+                                'id' => $user['encodeId'],
+                                'type' => 'users'
+                            ]
+                        ]
+                    ]
+                ]
+            ]);
+
+            $response = json_decode($I->grabResponse(), true);
+
+            $I->seeResponseCodeIsSuccessful(201);
+            $I->seeResponseContainsJson([
+                'data' => [
+                    'id' => $response['data']['id'],
+                    'type' => 'events',
+                    'attributes' => [
+                        'user_id' => $response['data']['attributes']['user_id'],
+                        'event' => $response['data']['attributes']['event'],
+                        'value' => '',
+                        'tag' => $response['data']['attributes']['tag'],
+                        'referrer' => $response['data']['attributes']['referrer'],
+                        'ip' => $response['data']['attributes']['ip'],
+                        'meta' => $response['data']['attributes']['meta'],
+                        'created_at' => $response['data']['attributes']['created_at']
+                    ],
+                    'relationships' => [
+                        'owner' => [
+                            'data' => [
+                                'id' => $response['data']['relationships']['owner']['data']['id'],
+                                'type' => 'users'
+                            ]
+                        ]
+                    ]
+                ]
+            ]);
+
+            $recordEvents = [
+                'temporary_user_id' => $temporaryUserId,
+                'user_id' => $userId,
+                'event' => $event,
+                'value' => '',
+                'tag' => $tag,
+                'referrer' => $referrer,
+                'ip' => $ip,
+                'owner_id' => $user['id'],
+                'created_at' => $date
+            ];
+
+            $I->seeRecord('events', $recordEvents);
+        }
+    }
+
+    public function createHundredSummarizableEvents(ApiTester $I)
```

Comment:
First of all, this method duplicates the previous line-to-line except for the count of the creating events. Then, you don't make any requests to the statistics, so there is no sense of tests like this.

Reply:
These tests are essential to check how statistics by events with summarizable type counts, not that statistics are written.